docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml

```diff
@@ -1692,6 +1694,63 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
 
         <?hard-pagebreak ?>
 
+        <para>
+          <emphasis role="bold">To add feature authentication flags to a
+          preexisting ECL Watch configuration:If your</emphasis>
+        </para>
+
+        <para>If you update the platform, but are using a preexisting
+        configuration, you could encounter a situation where Feature level
+        authentication flags do not exist. Missing flags can deny access to
```

Comment:
you could encounter a situation where Feature level authorization flags are not automatically created.